modules/_labs/panel/react/lib/Panel.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, spacing, CanvasSpacingValue} from '@workday/canvas-kit-react-core';
+
+export interface PanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Adjust padding for the contents of the Panel. Default: `spacing.s`
+   */
+  padding?: CanvasSpacingValue;
+  /**
+   * Determines the side in which the Panel can open from. Default: `PanelDirection.Right`
+   */
+  openDirection?: PanelDirection;
+  /**
+   * Sets the width of the Panel. Default: `360px`
+   */
+  width?: number;
+  /**
+   * Allows to either show or hide a drop shadow on the Panel. Default: `false`
+   */
+  showDropShadow?: boolean;
+  /**
+   * Optional to pass a PanelHeader component shows an optional string and close button
+   */
+  header?: React.ReactElement;
+}
+
+export enum PanelDirection {
+  Left,
+  Right,
+}
+
+const PanelContainer = styled('div')<
+  Pick<PanelProps, 'width' | 'showDropShadow' | 'openDirection'>
+>(
+  {
+    height: '100%',
+    backgroundColor: 'white',
+    display: 'flex',
+    flexDirection: 'column',
+    position: 'absolute',
```

Comment:
The design specs show this with rounded borders if it is "popped out" from the side of the page. How do we intend to support that use case? Is that future functionality to be added?